### PR TITLE
Upgrade versions

### PR DESCRIPTION
**Pull request by Diaphteiros**
**Release note**:

```improvement operator
Upgrade 'virtual' kube-apiserver to `v1.16.6`
```
```improvement operator
Upgrade kubernetes versions to the latest patch versions. From now on, for each minor version, only one patch version will be supported (unless required otherwise in special cases).
```
```improvement operator
Upgrade Gardener extension 'provider-aws' to `v1.3.3`
```

### CODE REVIEW

**Comment by vpnachev** [comment]
The virtual kube-apiserver cannot be upgraded to v1.16, ref https://github.com/gardener/gardener/issues/1605

**Comment by ialidzhikov** [comment]
> The virtual kube-apiserver cannot be upgraded to v1.16, ref [gardener/gardener#1605](https://github.com/gardener/gardener/issues/1605)

Yeah, nice catch.

**Comment by Diaphteiros** [comment]
Oh, thanks for the info. I'll go with the latest 1.15 version, then.

File: components/gardencontent/profiles/manifests/manifest.yaml [hunk 1]
```diff
@@ -48,13 +48,10 @@ defaults:
...
   kubernetes:
     versions:
-      - version: 1.17.1
-      - version: 1.16.4
-      - version: 1.16.3
-      - version: 1.15.7
-      - version: 1.15.6
+      - version: 1.17.2
+      - version: 1.16.6
+      - version: 1.15.9
```
Comment (vpnachev) [inline components/gardencontent/profiles/manifests/manifest.yaml]: Recently were released v1.17.3, v1.16.7 and v1.15.10
* https://github.com/kubernetes/kubernetes/releases/tag/v1.17.3
* https://github.com/kubernetes/kubernetes/releases/tag/v1.16.7
* https://github.com/kubernetes/kubernetes/releases/tag/v1.15.10

Comment (Diaphteiros) [reply components/gardencontent/profiles/manifests/manifest.yaml]: Thanks for the info, I'll update the commit.

**Pull request merged**